tests/func/test_add.py

```diff
@@ -513,13 +514,56 @@ def test(self):
 
 @pytest.mark.skipif(os.name != "nt", reason="Windows specific")
 def test_windows_should_add_when_cache_on_different_drive(
-    dvc_repo, repo_dir, temporary_windows_drive
+    tmp_dir, dvc, temporary_windows_drive
 ):
-    ret = main(["config", "cache.dir", temporary_windows_drive])
-    assert ret == 0
-
-    ret = main(["add", repo_dir.DATA])
-    assert ret == 0
+    tmp_dir.gen("file", "file content")
+
+    dvc.config.set("cache", "dir", temporary_windows_drive)
+    dvc = DvcRepo(dvc.root_dir)
```

Comment:
Do we need to rewrap it?

Reply:
We need to reload Cache config, changed it to be more precise

Reply:
Ok, let's leave it this way.